Utilities/Scripts/runCodespell.sh

```diff
@@ -45,5 +47,11 @@ als,\
 sinc,\
 therefor,\
 extracter,\
-inactivate\
+inactivate,\
+dependees,\
+pullrequest,\
+reson,\
+linez,\
+extit,\
+nnumber
```

Comment:
I didn't know how to best handle this. It was getting caught with strings such as "\nNumber" and wanting to make it "\number".

Reply:
This `\n` misinterpretation seems to be a known, open issue: https://github.com/codespell-project/codespell/pull/1422

I would not add an invalid word to the dictionary (it fixes one error but introduces chance for undetected errors). We could change the formatting, for example we could rewrite it to `"something\n" "number"`.

We could also change `extIt` to something else, too.

We can change `Magn Reson Imaging` to `Magnetic Resonance Imaging`.

We can change lineX, lineY, lineZ to line_X, line_Y, line_Z.